### PR TITLE
docs(es): Apr 25 - 27

### LINKED ISSUE #19: docs(es): Apr 25 - 27
Pending issues:

- https://github.com/vuejs/docs/commit/f95a4e9bd799013f953a9ebc2e39d550e39ae367
- https://github.com/vuejs/docs/commit/6eda606f5f3d9e0d7f1cb9af1d557a7a31d2b9cd
- https://github.com/vuejs/docs/commit/447f7b909f41c1b1adc8e0d732df88bbce64400b
- https://github.com/vuejs/docs/commit/a4df330726bc3943d5a0521d6fbb756a3033d2f9
- https://github.com/vuejs/docs/commit/198ac14d06a7749f07f608c6e1aab3f909a3aa1f
- https://github.com/vuejs/docs/commit/ff73f5bacafb25516151dd54869c4bee29d10a0a

### PR DESCRIPTION
**Pull request by mrcego**
fix #19 

cc @icarusgk @glopzel

### CODE REVIEW

File: .vitepress/theme/components/Banner.vue [hunk 1]
```diff
@@ -38,15 +49,42 @@ html:not(.banner-dismissed) {
   left: 0;
   right: 0;
   height: var(--vt-banner-height);
-  line-height: 0;
+  line-height: var(--vt-banner-height);
   text-align: center;
   font-size: 12px;
```
Comment (icarusgk) [inline .vitepress/theme/components/Banner.vue]: font size should be 13px

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "type": "module",
```
Comment (icarusgk) [inline package.json]: Why "module"?

**Review by icarusgk — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (mrcego) [reply package.json]: Due to ESM warning with Vite.js. Check deploy logs.

Comment (icarusgk) [reply package.json]: Oh, just checked. I think the error was coming from the line on `Banner.vue`
```js
import ref from 'vue'
```

Try deleting the `"type": "module"` to see if it triggers the same error.

Comment (mrcego) [reply package.json]: It was another error. Will try your suggestion.

**Comment by mrcego** [comment]
Ok, `type:module` is not relevant after all. Good to know that.

**Comment by mrcego** [comment]
> Ok, `type:module` is not relevant after all. Good to know that.

It's happening again @icarusgk 

image

**Comment by icarusgk** [comment]
I believe this time is happening because of a version change in the package.json that happened when merging `spanish-translation`. The version that throws this error is rc.31
Frame 223

**Comment by mrcego** [comment]
> I believe this time is happening because of a version change in the package.json that happened when merging `spanish-translation`. The version that throws this error is rc.31 Frame 223

But why it's happening this? My current branch is working with vitepress version according date range.

image

**Comment by icarusgk** [comment]
I guess that when merging the `spanish-translation` branch the vitepress version was overwritten and the CI ran with that version.

**Comment by icarusgk** [comment]
Just checked and it indeed ran with the latest commit to the branch, so maybe you could try to change the vitepress version back to .60 in one commit and set it back to .74 in another commit and then push those two commits, and see if that solves it.

**Comment by mrcego** [comment]
Hmmm... there is an issue here with CI while deploying and the same time, commit a change. Pipeline is setup for queue deploys?

**Comment by icarusgk** [comment]
I think it is 🤔

**Comment by mrcego** [comment]
Is maybe the issue happening for the vitepress alpha-74 version? 🧐

**Comment by mrcego** [comment]
Ok, tests are passing with alpha-60. When I set alpha-74, it is crashing and fallback to latest RC. full of weirdos.

**Comment by icarusgk** [comment]
that's weird, CI is bugging lol maybe it's using a cache. Good thing it's fixed now in the latest commit.

**Comment by icarusgk** [comment]
It's taking the `spanish-translation` version again, could you force a new deploy with the new version please. After merging this PR I hope it's gone.

**Comment by mrcego** [comment]
~~Nope, there is something wrong... I don't know if vitepress version exists on the npm BD and fallback, o definitely pipeline is caching installation.~~

It's fixed now @icarusgk

**Review by icarusgk — APPROVED** [review]

**Review by glopzel — APPROVED** [review]

**Pull request merged**